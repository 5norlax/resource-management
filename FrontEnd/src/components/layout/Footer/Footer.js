import React from 'react'

export default React.memo(function Footer() {
  return (
    <div>
      <hr />
      <p className="text-center">
        Copyright Â© 2019. Generic Logo Company. All Rights Reserved.
      </p>
    </div>
  )
})
