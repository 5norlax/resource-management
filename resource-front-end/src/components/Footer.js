import React from 'react';

const Footer = () => {
    return (
        <div>
            <hr />
            <p className="text-center">Copyright Â© 2019. YASH Technologies. All Rights Reserved.</p>
        </div>
        )
    }
    
export default Footer