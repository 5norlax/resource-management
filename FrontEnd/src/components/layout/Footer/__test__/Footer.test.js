import React from 'react';
import { render, cleanup } from '@testing-library/react';
import renderer from 'react-test-renderer';
import Footer from '../Footer';

afterEach(cleanup);

describe('Test Footer', () => {

  it('matches snapshot', () => {
    const tree = renderer.create(<Footer />).toJSON()
    expect(tree).toMatchSnapshot();
  });

});