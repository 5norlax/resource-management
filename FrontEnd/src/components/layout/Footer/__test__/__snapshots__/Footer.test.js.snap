// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Footer matches snapshot 1`] = `
<div>
  <hr />
  <p
    className="text-center"
  >
    Copyright Â© 2019. Generic Logo Company. All Rights Reserved.
  </p>
</div>
`;
